sympy/vector/coordsys.py

```diff
@@ -44,6 +54,7 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             Used for simple str printing.
 
         """
+        from sympy.abc import a, x, y, z
```

Comment:
> These should probably be dummies instead of public symbols.
I am not aware of the difference it would make. Can you specify any undesired consequences?